examples/riot-kitchen-sink/src/stories/__snapshots__/addon-actions.stories.storyshot

```diff
@@ -0,0 +1,243 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
these snapshots seem not right to me... probably because you are reusing the `rootElement` in a `renderTree` function

Reply:
should I discard "<root>" and "</root>" when comparing ? if yes, how ?

Reply:
<root> is not a problem. When I check the html of the `addon-actions` story I see this:

![image](https://user-images.githubusercontent.com/7867954/44780206-16228e00-ab8a-11e8-8e0d-915e1a13cacd.png)

How it comes the snapshot contains all these 👇 inside the button?

Reply:
all these what ?

Reply:
😃 
![image](https://user-images.githubusercontent.com/7867954/44780620-09eb0080-ab8b-11e8-8f69-eab367abbd18.png)

Reply:
Sorry I was reading the thread inside the conversation, not inside the diff

Reply:
ok

Reply:
thank you, I am going to investigate, even if I suspect quite easily the trouble
(among the things I don't like with riot, there is ths __TMPL_IMPL cache var which keeps every single template statically)

Reply:
Ok, found the issue,
I was almost never rendering something because I did not call the render-riot core method.
I had to make a bridge between storyshots and the core method (I wanted to stay DRY of course).
Unless if you have a better idea.